src/schema/geneInteraction.yaml

```diff
@@ -0,0 +1,467 @@
+    is_a: Entity
+    description: >-
+      A typed association between two entities, supported by evidence
+    comments:
+      - This is roughly the model used by biolink and ontobio at the moment
+    slots:
+      - subject
+      - predicate
+      - object
+      - relation
+      - negated
+      - qualifiers
+      - association_publications
+    slot_usage:
+      type:
+        description: rdf:type of biolink:Association should be fixed at rdf:Statement
+      category:
+        range: category type
+        required: false
+    exact_mappings:
+      - OBAN:association
+      - rdf:Statement
+      - owl:Axiom
+      - biolink:association
+
+  GeneToGeneAssociation:
+    description: >-
+      abstract parent class for different kinds of gene-gene or gene product
+      to gene product relationships. Includes homology and interaction.
+    abstract: true
+    is_a: Association
+    defining_slots:
+      - subject
+      - object
+    slot_usage:
+      subject:
+        range: Gene
+        description: >-
+          the subject gene in the association. If the relation is symmetric,
+          subject vs object is arbitrary. We allow a gene product to stand
+          as a proxy for the gene or vice versa.
+      object:
+        range: Gene
+        description: >-
+          the object gene in the association. If the relation is symmetric,
+          subject vs object is arbitrary. We allow a gene product to stand
+          as a proxy for the gene or vice versa.
+    exact_mappings:
+      - biolink:gene_to_gene_association
+
+  GeneInteraction:
+    description: >-
+      An interaction between two genes or gene products; this may be a physical
+      molecular interaction between gene products (e.g. protein-protein interactions),
+      or may be a genetic interaction between genes (e.g. phenotypic suppression)
+    is_a: GeneToGeneAssociation
+    abstract: true
+    slots:
+      cross_references
+      data_provider
+      interaction_type
+      interactor_A_role
+      interactor_B_role
+      interactor_A_type
+      interactor_B_type
+    defining_slots:
+      - subject
+      - predicate
+      - object
+    id_prefixes:
+      - biogrid
+      - dip
+      - emdb
+      - empiar
+      - FB
+      - imex
+      - intact
+      - mint
+      - pdbe
+      - rcsb pdb
+      - WB
+      - wwpdb
+    slot_usage:
+      id:
+        description: >-
+          The unique primary identifier for the interaction. This should be the source
+          (curation) database identifier, or if that is not available then the aggregation
+          database identifier (e.g. IMEx)
+      cross_references:
+        description: >-
+          Additional identifier(s) of the interaction annotation other than the primary id.
+      subject:
+        range: Gene                   # is this declaration needed since it is already made in the parent class?
+        values_from:
```

Comment:
What are you trying to say with "values_from" here?

Reply:
Just my attempt at indicating that genes put here should come from the Alliance. Wasn't quite sure how to use "values_from"